src/dotnet/commands/dotnet-remove/dotnet-remove-reference/Program.cs

```diff
@@ -42,10 +43,20 @@ internal class RemoveProjectToProjectReferenceCommand : CommandBase
         public override int Execute()
         {
             var msbuildProj = MsbuildProject.FromFileOrDirectory(new ProjectCollection(), _fileOrDirectory);
+            var references = _appliedCommand.Arguments.Select(p => {
+                var fullPath = Path.GetFullPath(p);
+                if (!Directory.Exists(fullPath)) {
+                    return p;
+                }
+                return Path.GetRelativePath(
```

Comment:
NIT: empty line after closing curly braces.

Reply:
Will fix.

Reply:
We need to put this in the editorconfig for the project or something like that, so that people don't have to "learn" these things.

Reply:
Pushed up the formatting change.